### PR TITLE
Bug fix to na_elementsw_account.py

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
na_elementsw_account wasn't working correctly and was a little complicated to use. We've simplified it and made it match the style of the existing modules. 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
- na_elementsw_account.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by lonico — APPROVED** [review]
shipit

**Review by thedoubl3j — APPROVED** [review]
bug fix, lgtm, shipit

File: lib/ansible/modules/storage/netapp/na_elementsw_account.py [hunk 1]
```diff
@@ -86,16 +86,35 @@
     username: "{{ elementsw_username }}"
     password: "{{ elementsw_password }}"
     state: present
+    status: locked
     element_username: TenantA
-    new_element_username: TenantA-Renamed
+
+- name: Rename Account
```
Comment (thedoubl3j) [inline lib/ansible/modules/storage/netapp/na_elementsw_account.py:92]: is there a difference between renaming and modifying? (line 101). I am not seeing any diff arguments

Comment (lonico) [reply lib/ansible/modules/storage/netapp/na_elementsw_account.py:92]: Yes.  There is a from_name for a rename.  In 101, I was asking for an action that would require a rename (from_name) and modify (status: locked) at the same time.

Comment (thedoubl3j) [reply lib/ansible/modules/storage/netapp/na_elementsw_account.py:92]: roger dodger. thanks for pointing out what my eyes missed @lonico

**Pull request merged**